src/assertions/AssertionGenerator.js

```diff
@@ -1,6 +1,8 @@
 import UnexpectedHtmlLike from 'unexpected-htmllike';
 import React from 'react';
+import { eventTypes } from 'react-dom/lib/SimpleEventPlugin';
```

Comment:
That's your call -- it was also my first instinct to hardcode it, but then I noticed that it was right there in that file.

But hmm, since `react-dom` is a peer dependency I agree that it could get messy real quick if that file is moved around inside the package. Let's hardcode it instead.